integration/Substra/README.md

```diff
@@ -0,0 +1,41 @@
+# Substra
+
+[Substra](https://docs.substra.org/en/latest/) is an open source federated learning framework. It provides a flexible python interface and a web app to run federated learning training at scale.
```

Comment:
```suggestion
[Substra](https://docs.substra.org/) is an open source federated learning framework. It provides a flexible python interface and a web app to run federated learning training at scale.
```

`latest` does not match any release, let's not redirect users to it
I expect this link to be functional in the next week

Reply:
Yes, I pointed it out in the PR description. I'm waiting for the release to fix the doc link